kalite/topic_tools/__init__.py

```diff
@@ -32,7 +32,7 @@
 from fle_utils.general import softload_json, json_ascii_decoder
 from kalite import i18n
 
-from kalite.main import models as main_models
+from kalite.topic_tools import models as main_models
```

Comment:
`main_models` seems like a misnomer now, and perhaps we can make this a relative import, also